src/openslide-vendor-dicom.c

```diff
@@ -57,6 +56,7 @@ struct dicom_file {
   DcmDataSet *file_meta;
   DcmDataSet *metadata;
   enum image_format format;
+  enum _openslide_jp2k_colorspace jp2k_colourspace;
```

Comment:
The enum (and indeed everything in OpenSlide) uses American spelling, so let's do that with the field name too.

Reply:
Ooop, sorry, an accident.